spec/BowlingSpec.js

```diff
@@ -0,0 +1,64 @@
+describe('Bowling Scorecard', function() {
+  var scorecard, frame;
```

Comment:
`scorecard` isn't used - this file is really `FrameSpec.js` (a unit test for `Frame`)